test/html2md.test.js

```diff
@@ -19,10 +19,10 @@ import { Nock } from './utils.js';
 
 async function test(spec) {
   const html = await readFile(resolve(__testdir, 'fixtures', `${spec}.html`), 'utf-8');
-  const expected = await readFile(resolve(__testdir, 'fixtures', `${spec}.md`), 'utf-8');
   const actual = await html2md(html, {
     log: console,
   });
+  const expected = await readFile(resolve(__testdir, 'fixtures', `${spec}.md`), 'utf-8');
```

Comment:
why did you move this?

Reply:
one of the added tests doesn't have an actual to compare, it should throw on the conversion